src/pages/Root/components/AppHeader/MobileAppHeader/MobileAppHeader.js

```diff
@@ -122,7 +122,6 @@ const Container = styled.header`
 	border-style: solid;
 	border-width: 1px 0;
 	position: fixed;
-	top: 0;
```

Comment:
In mobile the app banner causes the mobile app header to kinda break... having a "top: 0" here means the app banner will never be shown. The solution is to add a listener for the scroll event and adjust the top value based on that... but this can be done in a later PR.

Reply:
yeah I was wondering if we should actually make the banner absolute. On top of the header, with an opacity of like 0.6